### PR TITLE
qa: krbd whole-object-discard test

### PR DESCRIPTION
**Pull request by idryomov**
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>

### CODE REVIEW

File: qa/rbd/krbd_whole_object_discard.t [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  $ DEV=$(sudo rbd map cloneimg1)
+  $ hexdump $DEV
+  0000000 0000 0000 0000 0000 0000 0000 0000 0000
+  *
+  6400000 cdcd cdcd cdcd cdcd cdcd cdcd cdcd cdcd
+  *
+  c800000
+  $ sudo rbd unmap $DEV
+
+  $ rbd clone --stripe-unit 512K --stripe-count 7 img@snap cloneimg2
```
Comment (dillaman) [inline qa/rbd/krbd_whole_object_discard.t]: Nit: might be nice to have a quick comment about what you are attempting to test w/ these magic stripe counts

Comment (idryomov) [reply qa/rbd/krbd_whole_object_discard.t]: Just varying the object set size and asserting on the number of objects created.  I'll add the math for each clone.

**Comment by idryomov** [comment]
@dillaman Updated.

**Comment by idryomov** [comment]
http://pulpito.ceph.com/dis-2018-03-06_14:54:24-krbd-master-testing-basic-smithi/

**Pull request merged**